### DISCUSSION TITLE
Unable to boot rom

### DISCUSSION THREAD

**Discussion by David-figueiredo** [tjboldt/ProDOS-ROM-Drive#18] [Q&A]
Just got the rom but unable to boot. On slot 0 I get a green screen and in any other slot the attached screen 
![IMG_5566](https://github.com/tjboldt/ProDOS-ROM-Drive/assets/147448697/8e880e78-6c28-4709-b82d-3a46a52cab82)
Any help would be appreciated 
Tia

**Comment by tjboldt** [comment]
Slot 0 doesn’t have firmware or I/O space so that is expected. The odd characters are due to not having lowercase character ROM. Not booting might be if you have less than 64KB RAM.

**Comment by David-figueiredo** [comment]
Tks TJ already ordered a language card, let’s see if it works

On 11 Oct 2023, at 18:29, Terence Boldt ***@***.***> wrote:

﻿

Slot 0 doesn’t have firmware or I/O space so that is expected. The odd characters are due to not having lowercase character ROM. Not booting might be if you have less than 64KB RAM.
